IOMC/RandomEngine/python/IOMC_cff.py

```diff
@@ -169,4 +169,13 @@
 
 randomEngineStateProducer = cms.EDProducer("RandomEngineStateProducer")
 
-
+from Configuration.StandardSequences.Eras import eras
+if eras.phase2_muon.isChosen() or eras.phase2dev_muon.isChosen():
```

Comment:
I'm afraid this is not the appropriate way to use eras. We no longer want to support `isChosen()`. See https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideCmsDriverEras for documentation.